crates/schema/Cargo.toml

```diff
@@ -2,6 +2,8 @@
 name = "spacetimedb-schema"
 version.workspace = true
 edition.workspace = true
+license-file = "LICENSE"
+description = "Schema library for SpacetimeDB"
```

Comment:
@kazimuth feel free to improve this - I just put in some boilerplate text to get it to publish successfully